requirements/edx/base.txt

```diff
@@ -260,3 +260,6 @@ xblock==1.2.1
 zendesk==1.1.1
 django-tinymce==2.8.0
 django-hvad==1.8.0
+
+# UCDC SSO and API integration
+git+https://github.com/raccoongang/edx-oauth-client.git@ucdc-hawthorn-master#egg=edx_oauth_client
```

Comment:
I would add lib via pin to the stable tag instead of the branch it could be unstable.

Reply:
done.